students/Sigul4/hw-8/task-3.js

```diff
@@ -0,0 +1,13 @@
+// ********************
+// 
+// Search count of substr
+// 
+// ********************
+
+let str = 'Ехал Грека через реку, видит Грека в гречке рак!';
+
+function SearchSubstringAmount(initialString, searchingString){
+    return initialString.split(searchingString).length -1
```

Comment:
Function should return an amount of the searching string in the initial one **or an exception string 'There is not 'cat' in the initial string.' if is not such substring**